pkg/minikube/tunnel/route_linux.go

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
I was going to suggest golang.org/x/net/route - but it seems woefully incomplete for your usage :(

Reply:
yeah, I looked at syscall level integration as well, but it did not seemed simpler than calling out to netstat